### DISCUSSION TITLE
Providing basic preset workflows will helps usability.

### DISCUSSION THREAD

**Discussion by ltdrdata** [Comfy-Org/ComfyUI#189] [Ideas]
Currently, ComfyUI only offers the Default workflow, while the WebUI provides a range of features that are built-in. 

We can provide basic presets combo box such as img2img, i2i upscale, t2i with lora, i2i with lora, and more. 

This is a significant advantage for beginners who may experience difficulties when customizing workflows, which can be a barrier to their first experience. 

With these presets, users can easily select the feature they want without the need for complex explanations.

**Comment by WASasquatch** [comment]
I agree and disagree. xD The idea for a presets drop down, or a selector in settings is really good, but I don't think we should be comparing to WebUI. Node network breakdowns like this are supposed to be focused, and customized workflows. Not feature rich GUI.

Node networks are inherently really geared at power users / advance usage for things you know that you are doing (knowing the breakdown of all the steps you need and thus the relevant nodes to use). It sorta implies a bit of understanding of the mechanics being deployed.

**Reply by ltdrdata** [reply]
I concur that there is no need for us to adhere to the WebUI approach. We can provide convenient features in our own way.
While this may be geared towards experts, we should still provide a basic starting point to avoid starting from scratch. This will help to reduce the learning curve.

**Comment by WASasquatch** [comment]
To this effect:

@comfyanonymous do you think it's possible to split the node network top and bottom with GUI, this GUI expects certain functionality, and then a user inserts nodes in-between these "gateways" (to ensure the GUI functions). This is a extreme overhaul idea but kinda just curious. I use Terragen a lot, and I like that it's a mix between the two, a GUI, and a node network. 

![image](https://user-images.githubusercontent.com/1151589/228041065-9943ff4b-8ecf-4b0b-8c4f-d606995976ad.png)

Could be something to think about down the road, to garner users from both worlds into the ComfyUI world.

**Comment by rethink-studios** [comment]
But do you love Terragen's render times? :)

On Mon, Mar 27, 2023 at 2:03 PM WAS ***@***.***> wrote:

> To this effect:
>
> @comfyanonymous <https://github.com/comfyanonymous> do you think it's
> possible to split the node network top and bottom with GUI, this GUI
> expects certain functionality, and then a user inserts nodes in-between
> these "gateways" (to ensure the GUI functions). This is a extreme overhaul
> idea but kinda just curious. I use Terragen a lot, and I like that it's a
> mix between the two, a GUI, and a node network.
>
> [image: image]
> <https://user-images.githubusercontent.com/1151589/228041065-9943ff4b-8ecf-4b0b-8c4f-d606995976ad.png>
>
> Could be something to think about down the road, to garner users from both
> worlds into the ComfyUI world.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/comfyanonymous/ComfyUI/discussions/189#discussioncomment-5444825>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOKR6W5LMAPBAVMSHFBNTILW6HQBRANCNFSM6AAAAAAWBXMGQI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Brian Bullock

Founder / Director of CGI

RETHINK Studios

https://www.rethinkstudios.tv/
708-289-1059
***@***.***
https://ai.rethinkstudios.tv/
[image: linkedin] <https://www.linkedin.com/in/brian-bullock-9aa0512/>
[image: instagram] <https://www.instagram.com/brian.bullock.796/>

**Reply by WASasquatch** [reply]
That's why I got the i9 39000kf for that 5.8ghz boost xD I have one of the fastest times on the benchmark list. Like second or third. </off topic>
